Git Streak Tracker/SettingsView.swift

```diff
@@ -91,21 +134,34 @@ struct SettingsView: View {
                                 )
                                 .offset(x: -2, y: 1)
                             }
-                            else  {
+                            else {
                                 FontIcon.text(
                                     .ionicon(code: userStore.contributionData.error ? .md_close_circle : .md_checkmark_circle),
                                     fontsize: 12,
-                                    color: userStore.username.isEmpty ? .clear : userStore.contributionData.error  ? ColorPallete.midRed : ColorPallete.highlightGreen
+                                    color: getSpinnerColor()
```

Comment:
So this getSpinnerColor function seems to not even be used for the input spinner color atm, right? 😆  Maybe should either rename function or adapt it to actually handle the spinnerColor.

Reply:
Oh yeah duh